NuGet.Docs/App_Code/TopicHelper.cshtml

```diff
@@ -1,37 +1,62 @@
 ﻿@using NuGet.Docs
 @helper RenderDocumentationIndex()
 {
-    // Normalize the heading levels to start at 3 in the TOC.
-    const int minHeading = 3;
-    // Maximum depth of the index topic-tree
-    const int maxLevels = 3;
+// Normalize the heading levels to start at 3 in the TOC.
+const int minHeading = 3;
+// Maximum depth of the index topic-tree
+const int maxLevels = 3;
+
+var topics = Topic.GetTopicsWithSubTopics("~/ndocs");
+var orderedList = new List<Topic>();
+orderedList.Add(topics.Where(
+(a) => { return a.Title == "Quickstart"; }).First());
+orderedList.Add(topics.Where(
+(a) => { return a.Title == "Guides"; }).First());
+orderedList.Add(topics.Where(
+(a) => { return a.Title == "Create Packages"; }).First());
+orderedList.Add(topics.Where(
+(a) => { return a.Title == "Consume Packages"; }).First());
+orderedList.Add(topics.Where(
+(a) => { return a.Title == "Hosting Packages"; }).First());
+orderedList.Add(topics.Where(
+(a) => { return a.Title == "Tools"; }).First());
+orderedList.Add(topics.Where(
+(a) => { return a.Title == "Schema"; }).First());
+orderedList.Add(topics.Where(
+(a) => { return a.Title == "API"; }).First());
+orderedList.Add(topics.Where(
+(a) => { return a.Title == "Visual Studio Extensibility"; }).First());
+orderedList.Add(topics.Where(
+(a) => { return a.Title == "Policies"; }).First());
+    @RenderIndexTopics(orderedList, minHeading, minHeading + maxLevels);
 
-    var topics = Topic.GetTopicsWithSubTopics("~/")
-        .Where(t => t.SubTopics.Any()
-            && !String.Equals(t.Title.ToLowerInvariant(), "release notes"))
-        .OrderBy(t => t.Title.ToLowerInvariant() == "consume" ? 1 : t.Title.ToLowerInvariant() == "create" ? 2 : 3);
-    @RenderIndexTopics(topics, minHeading, minHeading + maxLevels);
 }
 
 @helper RenderIndexTopics(IEnumerable<Topic> topics, int level, int maxDepth)
 {
-    foreach (Topic topic in topics)
+foreach (Topic topic in topics)
+{
+    string title = @topic.Title.Replace(" ", "-");
+        <li id="a-@title" class="level1container"/>
```

Comment:
Ok but it is closed twice, here with the `/>` and [here](https://github.com/NuGet/NuGetDocs/pull/572/files/9cec6e159b6c189fd678a079e5d71c87a5dbf722#diff-f25072f9e3b5f3f0b2264b83ce78866dR50) with a proper end tag. One of these should be removed and I think it is this one :-)